src/test/resources/org/broadinstitute/hellbender/tools/diagnostics/expected_PrintBGZFBlockInformationIntegrationTest_testNormalInput.out

```diff
@@ -0,0 +1,1712 @@
+Block at file offset 0
```

Comment:
Perhaps these output files should have a header pointing to the file they are summarizing to avoid confusion?

Reply:
Good idea, added.